app-shells/mksh/mksh-54.recipe

```diff
@@ -36,7 +36,6 @@ REQUIRES="
 BUILD_REQUIRES="
 	haiku_devel
 	"
-
```

Comment:
Since the only change in `mksh-54.recipe` is the removal of this empty line then I would recommend to drop this change from this PR and find another recipe that needs a more consistent fix ;-)
(Next time mksh gets a new release we will remove the empty line.)